### ISSUE TITLE
Mobile: blank area at the bottom and offset on the parallaxes

### ISSUE THREAD

**Issue by nagman** [Prinzhorn/skrollr#782]
Hi,

I've created a parallax page for desktop and mobile (based on your [kitties classic demo page](http://prinzhorn.github.io/skrollr/examples/classic.html)) but I run into two issues on mobile display.

The first one is a high blank area at the bottom of the page (and when I set forceHeight to false, the scroll is disabled).

The second one is that all my parallaxes are shifted about 50px lower relative to the container.
When I test my page in Chrome mobile mode (on desktop), it works perfectly (except for the blank area at the bottom).

Here's the [page](http://armara.adelios.fr).

Any ideas?

**Comment by nagman** [comment]
Ok,

If anyone gets there for the same issue, I've solved it:

The skrollr.init was launched before every image in the page has loaded (so when the images loaded, the document was taller and the skrollr used the previous calculated height).

To get rid of the blank area at the bottom of the page, I just added <div data-bottom-top="line-height:0"></div> (I could set something else than line-height, this is not important). So that skrollr creates an empty element which forces the page to go to the very end.
